lightning_pose/data/datasets.py

```diff
@@ -350,32 +325,23 @@ def compute_heatmaps(self):
             size=(len(self.image_names), self.num_keypoints, *self.output_shape)
         )
         for idx in range(len(self.image_names)):
-            example_dict: BaseExampleDict = super().__getitem__(idx)
+            example_dict: BaseLabeledExampleDict = super().__getitem__(idx)
             label_heatmaps[idx] = self.compute_heatmap(example_dict)
 
         self.label_heatmaps = label_heatmaps
 
-    @typechecked
-    def __getitem__(self, idx: int) -> HeatmapExampleDict:
+    def __getitem__(self, idx: int) -> HeatmapLabeledExampleDict:
         """Get an example from the dataset.
 
         Calls the base dataset to get an image and a label, then additionally
         returns the corresponding heatmap.
 
         """
-        example_dict: BaseExampleDict = super().__getitem__(idx)
+        example_dict: BaseLabeledExampleDict = super().__getitem__(idx)
         if len(self.imgaug_transform) == 1 and isinstance(self.imgaug_transform[0], iaa.Resize):
             # we have a deterministic resizing augmentation; use precomputed heatmaps
             example_dict["heatmaps"] = self.label_heatmaps[idx]
         else:
             # we have a random augmentation; need to recompute heatmaps
             example_dict["heatmaps"] = self.compute_heatmap(example_dict)
         return example_dict
-
-    def get_fully_labeled_idxs(self):  # TODO: make shorter
```

Comment:
we should keep something like that as a util, at least for pca. is this the case?